tensorflow/core/util/batch_util.cc

```diff
@@ -156,6 +156,8 @@ Status CopySliceToElement(const Tensor& parent, Tensor* element, int64 index) {
   switch (parent.dtype()) {
     TF_CALL_ALL_TYPES(HANDLE_TYPE);
     TF_CALL_QUANTIZED_TYPES(HANDLE_TYPE);
+    TF_CALL_uint32(HANDLE_TYPE);
+    TF_CALL_uint64(HANDLE_TYPE);
```

Comment:
`batch_util::MaybeMoveSliceToElement` is also missing these cases, I didn't add them here because I haven't verified it's ok

Reply:
You can go ahead and add them there too. Thanks.

Reply:
You can verify that this works by adding a test to `unbatch_test.py` that uses uint32 / uint64 elements.

Reply:
sure, will do

Reply:
done